IQKeyboardManagerSwift/IQKeyboardManager.swift

```diff
@@ -853,8 +853,10 @@ public class IQKeyboardManager: NSObject, UIGestureRecognizerDelegate {
                     unwrappedController.view.layoutIfNeeded()
                 }
  
-                
                 }) { (animated:Bool) -> Void in}
+            
+            let notification = frame.origin.y == 0 ? IQKeyboardManagerWillReturnRootFrameToOriginNotification : IQKeyboardManagerWillMoveRootFrameFromOriginNotification
```

Comment:
imho, id rather shorten this as name, not notification. but this is a very small issue.